src/frontend/src/pages/AdminToolsPage/FinanceConfig/AccountCodeFormModal.tsx

```diff
@@ -65,6 +75,23 @@ const AccountCodeFormModal = ({ showModal, handleClose, defaultValues, onSubmit
         <FormLabel>Account Name</FormLabel>
         <ReactHookTextField name="name" control={control} fullWidth />
       </FormControl>
+      <FormControl fullWidth>
+        <FormLabel>Refund Source</FormLabel>
+        <Select
+          name="allowedRefundSources"
+          multiple
+          value={refundSourcesMultiSelect}
+          onChange={handleChange}
+          renderValue={(selected) => selected.join(', ')}
+        >
+          {refundSourcesMultiSelect.map((account) => (
+            <MenuItem key={account} value={account}>
+              <Checkbox checked={refundSourcesMultiSelect.indexOf(account) > -1} />
+              <ListItemText primary={account} />
+            </MenuItem>
+          ))}
+        </Select>
+      </FormControl>
```

Comment:
ok so I know we told you to use a select, but I think the select component is kinda weird upon looking at it closer, so let's just have you use a multi autocomplete (because why not). You should look at the blockedBy input on the work package edit form for an example of this (it should hopefully be mostly one to one). The core part of this I want you to use is that we're wrapping the autocomplete in a controller and using the value from the controller to set the value in the autocomplete and calling the onChange from the controller in the onchange method you define for the select. The other important part is defining a set of options with a label and and id. In your case the id will be of type ClubAccount rather than of int because the value we're saving is a ClubAccount not an int